validate.py

```diff
@@ -3,6 +3,7 @@
 
 import argparse
 import re
+import subprocess
```

Comment:
```suggestion
import subprocess
from typing import Optional
```
Goes along with changes down below.